index.php

```diff
@@ -127,19 +131,91 @@
+                                    <img src="https://a.ppy.sh/1" class="comments__pfp">
                                 </div>
-                                <div class="osekai__section-header-right">
-                                    <h3>gotta love it</h3>
-                                    <p>right?... right???</p>
+                                <div class="comments__post-right">
+                                    <div class="comments__post-upper">
+                                        <textarea rows="1" placeholder="Post a comment!"></textarea>
+                                    </div>
+                                    <div class="comments__post-lower">
+                                        <div class="comments__post-lower-right">
+                                            <p>osekai supports bbcode!</p>
+                                            <div class="comments__post-button"><i class="fas fa-plane"></i> Post</div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="osekai__divider"></div>
+                            <div class="comments__list">
+                                <div class="comments__comment">
+                                    <div class="comments__comment-left">
+                                        <img src="https://a.ppy.sh/1" class="comments__pfp">
+                                        <div class="comments__comment-votes">
+                                            +22
+                                        </div>
+                                    </div>
+                                    <div class="comments__comment-right">
+                                        <div class="comments__comment-content">
+                                            <div class="comments__comment-content-username">
+                                                <p class="comments__username">Username</p>
+                                                <div class="comments__comment-content-username-right">
+                                                    <p>on the right</p>
+                                                </div>
+                                            </div>
+                                            <div class="comments__comment-content-text">
+                                                <p>look at me im an awesome comment</p>
+                                            </div>
+                                        </div>
+                                        <div class="comments__comment-infobar">
+                                            <div class="comments__comment-infobar-info">
+                                                <i class="fas fa-calendar-alt"></i>
+                                                <p>posted <strong>5 days ago</strong></p>

```

Comment:
Should add a tooltip for localised time of comment.

Reply:
![image](https://github.com/Osekai/osekai-wip/assets/33783503/610b307b-1bda-4e6c-81bb-c96fcb220ba0)
there is